libcudacxx/include/cuda/std/detail/libcxx/include/cmath

```diff
@@ -685,7 +685,7 @@ __constexpr_isfinite(_A1 __lcpp_x) noexcept
     return isfinite(__lcpp_x);
 }
 
-#if defined(_MSC_VER) || defined(__CUDACC_RTC__) || defined(_CCCL_CUDA_COMPILER_CLANG)
+#if defined(_CCCL_COMPILER_MSVC) || defined(_CCCL_COMPILER_NVRTC)
```

Comment:
Just in case it was an oversight: This change removed `|| defined(_CCCL_CUDA_COMPILER_CLANG)`, but in further changes below, you left it.

Reply:
That is correct. For whatever reason `ptxas` does not find the function definitions of `logb` and `scalbn` if we use the builtin

Reply:
```
ptxas fatal   : Unresolved extern function 'scalbn'
clang: error: ptxas command failed with exit code 255 (use -v to see invocation)
Ubuntu clang version 16.0.6 (++20231112100510+7cbf1a259152-1~exp1~20231112100554.106)
Target: x86_64-pc-linux-gnu
Thread model: posix
InstalledDir: /usr/bin
clang: note: diagnostic msg: 
********************
```

Reply:
Alright, thx for confirming!